lang/erlang/patches/010-openssl-deprecated.patch

```diff
@@ -1,50 +1,37 @@
+From e734cd9e84c345bddb26ebe99ee5488ae0e45e01 Mon Sep 17 00:00:00 2001
+From: Xiaobo Tian <peterwillcn@gmail.com>
+Date: Tue, 21 Apr 2020 20:16:56 +0800
+Subject: [PATCH] openssl-deprecated
```

Comment:
Why do we need to have this patch? Was it upstreamed? If not, do it first. In your commit, there isn't anything related to this patch. There is just update and nothing else.